submissions/BlueLamantine/friends app/js/main.js

```diff
@@ -0,0 +1,329 @@
+                <span>first name</span>
+                <div class="first-name">${profileData.name.first}</div>
+              </div>
+              <div class="name-info block">
+                <span>last name</span>
+                <div class="last-name">${profileData.name.last}</div>
+              </div>
+            </div>
+            <div class="details">
+              <div class="details-info block">
+                <span>alias</span>
+                <div class="alias">V</div>
+              </div>
+              <div class="details-info block">
+                <span>years old</span>
+                <div class="age">${profileData.dob.age}</div>
+              </div>
+              <div class="details-info block">
+                <span>gender</span>
+                <div class="gender">${profileData.gender}</div>
+              </div>
+            </div>
+            <div class="social">
+              <div class="social-info block">
+                <span>phone</span>
+                <div class="tel">
+                  <a href="tel:+${profileData.phone}">+${profileData.phone}</a>
+                </div>
+              </div>
+              <div class="social-info block">
+                <span>email</span>
+                <div class="mail">
+                  <a href="mailto:${profileData.email}" class="user-mail">${profileData.email}</a>
+                </div>
+              </div>
+            </div>
+            <div class="location">
+              <div class="location-info block">
+                <span>country</span>
+                <div class="country">${profileData.location.country}</div>
+              </div>
+              <div class="location-info block">
+                <span>city</span>
+                <div class="city">${profileData.location.city}</div>
+              </div>
+            </div>
+          </div>
+          <div class="auth">
+              <span>authetication code</span>
+              <div class="code">
+                <img src="img/code.png" alt="User code" class="profile-code">
+              </div>
+          </div>
+        </div>
+      </div>
+    `;
+  }
+
+  getProfiles() {
+    this.data.forEach((result, index) => {
+      this.profilesContainer.insertAdjacentHTML(
+        'beforeend',
+        this.getHtmlProfile(result, index)
+      );
+    });
+  }
+
+  getHtmlFilter() {
+    return `   
+      <div class="filter filter-gender">
+        <p>filter by gender</p>
+        <div class="radio-wrapper">
+        <input type="radio" id="male" name="sort-gender">
+        <label for="male" class="radio-lable">Male</label>
+        <input type="radio" id="female" name="sort-gender">
+        <label for="female" class="radio-lable">female</label>
+        <input type="radio" id="allGender" name="sort-gender">
+        <label for="allGender" class="radio-lable">all</label>
+        </div>
+      </div>
+
+      <div class="filter filter-name">
```

Comment:
Maybe you should wrap it into `fieldset`?

Reply:
thanks for a tip, now it is ```<form>``` with ```fieldsets```